pkg/apis/core/types.go

```diff
@@ -2474,6 +2480,12 @@ type PodSpec struct {
 	// configuration based on DNSPolicy.
 	// +optional
 	DNSConfig *PodDNSConfig
+	// If specified, all readiness gates will be evaluated for pod readiness.
+	// A pod is ready when all its containers are ready AND
+	// all conditions specified in the readiness gates have value equal to "True"
```

Comment:
It seems like we are using `PodConditionType` for `ReadinessGate`, and "True" is not one of the valid values?

Reply:
I should say the status of corresponding condition should be "True"

Reply:
Ah got it, wasn't aware that condition can have status :)